chips/apollo3/src/lib.rs

```diff
@@ -75,6 +75,7 @@ pub static IRQS: [unsafe extern "C" fn(); 32] = [CortexM4::GENERIC_ISR; 32];
 pub static PATCH: [unsafe extern "C" fn(); 16] = [unhandled_interrupt; 16];
 
 #[cfg(all(target_arch = "arm", target_os = "none"))]
+#[inline(always)]
```

Comment:
Why are we now inlining this?

Can you please comment to explain? We're learning that we can't keep track of why we are including inline directives on various functions, and we need to start documenting them.

Reply:
Comments added!